tests/main/lxd/task.yaml

```diff
@@ -1,5 +1,10 @@
 summary: Ensure that lxd works
 
+details: |
```

Comment:
There's a conflict here because a description was added recently

Reply:
Merged master in https://github.com/snapcore/snapd/pull/13333/commits/8fce545c70d2dc6928c9322680f030628a2e46b3 to resolve this.